src/adapter/src/coord/ddl.rs

```diff
@@ -697,10 +698,14 @@ impl Coordinator {
                                         )
                                     })?;
 
+                                let should_wait = match connection.flavor {
+                                    PostgresFlavor::Vanilla => true,
+                                    PostgresFlavor::Yugabyte => false,
```

Comment:
can we leave a comment explaining why this is necessary?

Reply:
Ah, yes. It just doesn't support `WAIT`. Will add it

Reply:
done